browser/src/map/handler/Map.Keyboard.js

```diff
@@ -136,15 +138,89 @@ L.Map.Keyboard = L.Handler.extend({
 		39  : true, // right arrow
 		40  : true, // down arrow
 		45  : true, // insert
-		113 : true  // f2
+		113 : true  // f2sss
 	},
 
+
 	keyCodes: {
-		pageUp:   33,
-		pageDown: 34,
-		enter:    13
+		BACKSPACE   : UNOKey.BACKSPACE,
```

Comment:
Please DO NOT USE `UNOKey` enum here. I said it was misleading comment from my side to show that dictionary as an example.

What we want here is to bind NAME with NUMBER. `pageUp: 33` and others should stay here, they are correct bindings. Please only add new ones for missing keyCodes used in this file.

like on the bottom we have `45`, you should add here: `INSERT: 45,`, then replace occurrences of `45` with `this.keyCodes.INSERT`  - this is done great already :)

Reply:
( `UNOKey` stores values like `1034` which are different than keyCodes used by the browser, `UNOKey` enum is for our server which uses different coding )